src/main/java/com/exasol/releasedroid/main/ReleaseDroid.java

```diff
@@ -88,18 +88,26 @@ private void validatePlatformNames(final List<PlatformName> platformNames) {
     }
 
     private void validateUserInput(final UserInput userInput) {
+        validateOwner(userInput);
+        validateGoal(userInput);
+        validateRepositoryName(userInput);
+        validateGoalAndBranch(userInput);
+        validateLocalPath(userInput);
+    }
+
+    private void validateOwner(final UserInput userInput) {
         if (!userInput.hasOwner()) {
             userInput.setOwner(EXASOL_REPOSITORY_OWNER);
         }
-        validateMandatoryParameters(userInput);
-        validateGoalAndBranch(userInput);
-        validateLocalPath(userInput);
     }
 
-    private void validateMandatoryParameters(final UserInput userInput) {
+    private void validateGoal(final UserInput userInput) {
         if (!userInput.hasGoal()) {
-            throwExceptionForMissingParameter("goal");
+            userInput.setGoal(Goal.VALIDATE);
         }
```

Comment:
We can rename these two method names, I am not sure if they are validating anymore. Maybe `checkGoal` and `checkOwner`?

Reply:
Done!